WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -9,9 +10,13 @@ def bad_api_parameter(e)
 
   DEFAULT_API_RESULT_LIMIT = 20
 
+  def current_resource_owner
```

Comment:
That was an accidental rejection of the safe navigation rewrite. I've rewritten this in https://github.com/jfly/worldcubeassociation.org/commit/4070442c038215faabd9442591fb99bd8091f68c.